src/content/reference/react/useImperativeHandle.md

```diff
@@ -36,25 +36,25 @@ const MyInput = forwardRef(function MyInput(props, ref) {
 
 [See more examples below.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `ref`: The `ref` you received as the second argument from the [`forwardRef` render function.](/reference/react/forwardRef#render-function)
+* `ref`: *ref* yang Anda terima sebagai argumen ke-dua dari [fungsi *render* `forwardRef`.](/reference/react/forwardRef#render-function)
```

Comment:
```suggestion
* `ref`: *ref* yang Anda terima sebagai argumen kedua dari [fungsi *render* `forwardRef`.](/reference/react/forwardRef#render-function)
```

It's either `ke-2` or `kedua`, not `ke-dua`.
[Referensi](https://ejaan.kemdikbud.go.id/eyd/penulisan-kata/angka-dan-bilangan/#9-penulisan-bilangan-tingkat-dapat-menggunakan-angka-romawi-gabungan-awalan-ke-dan-angka-arab-atau-huruf):
![image](https://user-images.githubusercontent.com/6315466/235990592-a1ababc8-31a1-47be-89d4-83ee058a8f67.png)